.github/ISSUE_TEMPLATE/defect-report.yml

```diff
@@ -0,0 +1,134 @@
+name: Defect Report
+description: File a defect report
+labels: [ "defect", "in triage" ]
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for helping us in making Dependency-Track better!
+      
+      **Is the defect you'd like to report only UI-related?**
+      We have a dedicated repo for the frontend, please create an issue [there](https://github.com/DependencyTrack/frontend/issues/new/choose)!
+      Unsure? That's fine, just report it here! The maintainers will migrate the issue if necessary.
+      
+      **Please do not ask questions here!**
+      If you're not sure whether what you're experiencing is in fact a defect,
+      you can reach out to maintainers and the broader community via various channels.
+      Refer to our [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#asking-questions) to find out where and how to ask questions.
+- type: textarea
+  id: behavior-current
+  attributes:
+    label: Current Behavior
+    description: |-
+      Describe the current faulty behavior that you observed.
+      Consider providing screenshots, log output, and other supplementary data.
+      
+      *Files and images can be included via drag and drop into this text field.*
+  validations:
+    required: true
+- type: textarea
+  id: steps-to-reproduce
+  attributes:
+    label: Steps to Reproduce
+    description: |-
+      Describe the exact steps of how the defect can be reproduced.
+      Consider providing screenshots, BOM files, and other supplementary data.
+      
+      *Files and images can be included via drag and drop into this text field.
+      For BOM files, please redact any internal or otherwise confidential information.*
+    value: |-
+      1.
+  validations:
+    required: true
+- type: textarea
+  id: behavior-expected
+  attributes:
+    label: Expected Behavior
+    description: >-
+      Describe how you expect Dependency-Track to behave instead.
+  validations:
+    required: true
+- type: markdown
+  attributes:
+    value: |
+      ## Environment
+      Please provide some details about the environment in which you observed the defect.
+- type: dropdown
+  id: environment-dtrack-version
+  attributes:
+    label: Dependency-Track Version
+    description: >-
+      What version of Dependency-Track are you using?
+    options:
+    - 3.8.x
+    - 4.0.x
+    - 4.1.x
+    - 4.2.x
+    - 4.3.x
+    - 4.4.x
+    - 4.5.0
+    - 4.6.0-SNAPSHOT
```

Comment:
Needs to be updated,  now that 4.6.0 has been released

Reply:
Done!